GlobalCache.php

```diff
@@ -17,7 +17,8 @@
 // This should ideally be patched upstream, converting the hook used
 // to SidebarBeforeOutput rather than SkinBuildSidebar, which is
 // more appropriate for this extension.
-if ( !$wi->config->get( 'wmgUseGroupsSidebar', $wi->dbname ) ) {
+// Disabling Sidebarcahce per wiki as a temporary solution (see T8732)
```

Comment:
A solution to what this doesn't explain much. Also there is a typo. Ideally I'd rather this be an entirely different if statement for code standards, but I'm not to big on it, so up to you.

Reply:
Well that's interesting, one MW Engineer wants it separate and one wants it together.

Reply:
Oh I didn't realise there was something else. To me it just makes more sense to be separate for better explanation of reasoning. The comments don't go good combined like that in my opinion.